src/main/java/mate/academy/model/Order.java

```diff
@@ -0,0 +1,75 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.Table;
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Entity
+@Table(name = "orders")
+public class Order {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @OneToMany(fetch = FetchType.LAZY)
```

Comment:
The `@OneToMany` annotation for `tickets` should include a `mappedBy` attribute to specify the owning side of the relationship. This is necessary for bidirectional relationships. For example, `@OneToMany(mappedBy = "order", fetch = FetchType.LAZY)` assuming `order` is the field in the `Ticket` class that maps back to `Order`.